### PR TITLE
Cleanup docs

### PR DESCRIPTION
**Pull request by ashfire908**
Currently a WIP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ require 'fauna'
 
 ### Creating a Client
 
-All API requests pass through a `Fauna::Client` Creating a client
-requires either a token, a server key, or a client key.
+All API requests pass through a `Fauna::Client`. Creating a client
+requires either a token or an admin, server, or client key.
```
Comment (freels) [inline README.md]: put token last

**Comment by freels** [comment]
shipit

File: README.md (hunk 1 above)
Comment (ashfire908) [inline README.md]: `either an admin, server, or client key, or a token` or `either an admin key, server key, client key, or a token`?

Comment (freels) [inline README.md]: The latter sounds clearer to me.

**Pull request merged**